production.ini

```diff
@@ -454,7 +454,7 @@ fedora_epel.mandatory_days_in_testing = 14
 ##
 
 # FAS Groups that we want to pay attention to
-# When a user logs in, bodhi will look for any of these groups and associate #
+# When a user logs in, bodhi will look for any of these groups and associate
```

Comment:
I'm not sure what the story is here... It was part of the original. It isn't technically a spelling fix, I can certainly split this off if requested...